tutorials/Working_with_fNIRS_data.py

```diff
@@ -0,0 +1,123 @@
+"""
```

Comment:
change filename to `plot_fnirs_data.py`, and add it to the `documentation.rst` somewhere.